sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-import-csv.service.ts

```diff
@@ -20,8 +20,18 @@ import { CustomNamedError } from "@sims/utilities";
 import { OFFERING_VALIDATION_CSV_PARSE_ERROR } from "../../constants";
 import { LoggerService, InjectLogger } from "@sims/utilities/logger";
 
+interface InstitutionDetails {
+  operatingName: string;
+  legalOperatingName: string;
+  primaryEmail: string;
+}
+interface LocationDetails {
+  id: number;
+  name: string;
+  institutionDetails: InstitutionDetails;
+}
```

Comment:
Not a blocker, for an object to be consumed internally we could easily keep it simple and use just a flat object.